CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+cmake_minimum_required(VERSION 3.8)
+cmake_policy(VERSION 3.8)
+
+###############################################
+# NOTE FOR THOSE UNFORTUNATES WHO DON'T KNOW OF HUNTER
```

Comment:
This seems like a _lot_ of overkill for just getting Boost. Not to mention that "those unfortunates" is a rather condescending tone to take. I get the impression from this text that you just like hunter a lot and as a result it's become your hammer for which every problem needs to be a nail.

Could you explain why `find_package(Boost REQUIRED ${MIN_VERSION})` isn't enough? That approach will work with a system-installed boost, which I doubt this Hunter thing does as from the little info I've gleaned about it it's basically it's own package manager.